From: Matthias Hopf <mhopf@suse.de>
Date: Wed, 18 Oct 2006 12:25:04 +0000 (+0200)
Subject: Fix double open of compose file.
X-Git-Url: http://gitweb.freedesktop.org/?p=xorg/lib/libX11.git;a=commitdiff;h=686bb8b35acf6cecae80fe89b2b5853f5816ce19

Fix double open of compose file.

Issue found by Kees Cook <kees@canonical.com>.
---

--- a/modules/im/ximcp/imLcIm.c
+++ b/modules/im/ximcp/imLcIm.c
@@ -617,7 +617,6 @@ _XimCreateDefaultTree(
     }
 #endif
 
-    fp = _XFopenFile (name, "r");
     if (! (fp = _XFopenFile (name, "r"))) {
 	if (tmpcachedir)
 	    Xfree  (tmpcachedir);
